app/main.py

```diff
@@ -1,8 +1,38 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
```

Comment:
Writing each parameter from a new line is a best practice

Reply:
@ValeriiaKyr this comment is not fixed. Parameters should start from the next line after method name